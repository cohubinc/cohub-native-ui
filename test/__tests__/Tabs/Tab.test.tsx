import React from "react";
import { shallow } from "enzyme";

import { Tab } from "dist";

it("renders without crashing", async () => {
  const wrapper = shallow(<Tab useRedux={false} title="test" />);

  expect(wrapper).toMatchSnapshot();
});
