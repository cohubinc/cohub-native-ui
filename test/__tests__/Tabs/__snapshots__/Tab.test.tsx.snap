// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="Tab-module_Tab__3boJB false undefined"
  onClick={[Function]}
>
  test
</div>
`;
